[core]/es_extended/server/main.lua

```diff
@@ -628,16 +628,21 @@ ESX.RegisterServerCallback('esx:getPlayerNames', function(source, cb, players)
   cb(players)
 end)
 
-ESX.RegisterServerCallback("esx:spawnVehicle", function(source,cb,model,coords,heading,props,warp)
+ESX.RegisterServerCallback("esx:spawnVehicle", function(source,cb,vehData)
   local ped = GetPlayerPed(source)
```

Comment:
Please handle if vehData is nil and vehData is table (we accept only table type) cb false and add log how "VehData is nil, please add vehData then parameters".